src/main/java/racingcar/RacingGameCore.java

```diff
@@ -0,0 +1,53 @@
+package racingcar;
+
+import type.MessageFactory;
+
+import java.util.ArrayList;
+
+class RacingGameCore {
+    private static final String INITIAL_POSITION_VALUE = "";
+
+    private final RacingCars racingCars;
+
+    private int stageCount;
+
+    private RacingGameCore(final String[] carNames, final int stageCount) {
+        ArrayList<Car> stringArrayList = new ArrayList<>();
```

Comment:
```suggestion
        List<Car> cars = new ArrayList<>();
```

이펙티브 자바 `아이템 64. 객체는 인터페이스를 사용해 참조하라`를 참고해주세요!

또한 stringArrayList라는 네이밍엔 

1. string이 아니다.
2. 자료형을 나타내고 있다.

라는 문제가 존재하고 있네요 🤔